dds/DCPS/XTypes/DynamicData.h

```diff
@@ -0,0 +1,217 @@
+  DDS::ReturnCode_t get_uint8_values(UInt8Seq& value, MemberId id) const;
+  DDS::ReturnCode_t set_uint8_values(MemberId id, const UInt8Seq& value);
+
+  typedef Sequence<ACE_CDR::Short> Int16Seq;
+  DDS::ReturnCode_t get_int16_values(Int16Seq& value, MemberId id) const;
+  DDS::ReturnCode_t set_int16_values(MemberId id, const Int16Seq& value);
+
+  typedef Sequence<ACE_CDR::UShort> UInt16Seq;
+  DDS::ReturnCode_t get_uint16_values(UInt16Seq& value, MemberId id) const;
+  DDS::ReturnCode_t set_uint16_values(MemberId id, const UInt16Seq& value);
+
+  typedef Sequence<ACE_CDR::LongLong> Int64Seq;
+  DDS::ReturnCode_t get_int64_values(Int64Seq& value, MemberId id) const;
+  DDS::ReturnCode_t set_int64_values(MemberId id, const Int64Seq& value);
+
+  typedef Sequence<ACE_CDR::ULongLong> UInt64Seq;
+  DDS::ReturnCode_t get_uint64_values(UInt64Seq& value, MemberId id) const;
+  DDS::ReturnCode_t set_uint64_values(MemberId id, const UInt64Seq& value);
+
+  typedef Sequence<ACE_CDR::Float> Float32Seq;
+  DDS::ReturnCode_t get_float32_values(Float32Seq& value, MemberId id) const;
+  DDS::ReturnCode_t set_float32_values(MemberId id, const Float32Seq& value);
+
+  typedef Sequence<ACE_CDR::Double> Float64Seq;
+  DDS::ReturnCode_t get_float64_values(Float64Seq& value, MemberId id) const;
+  DDS::ReturnCode_t set_float64_values(MemberId id, const Float64Seq& value);
+
+  typedef Sequence<ACE_CDR::LongDouble> Float128Seq;
+  DDS::ReturnCode_t get_float128_values(Float128Seq& value, MemberId id) const;
+  DDS::ReturnCode_t set_float128_values(MemberId id, const Float128Seq& value);
+
+  typedef Sequence<ACE_CDR::Char> CharSeq;
+  DDS::ReturnCode_t get_char8_values(CharSeq& value, MemberId id) const;
+  DDS::ReturnCode_t set_char8_values(MemberId id, const CharSeq& value);
+
+  typedef Sequence<ACE_CDR::WChar> WCharSeq;
+  DDS::ReturnCode_t get_char16_values(WCharSeq& value, MemberId id) const;
+  DDS::ReturnCode_t set_char16_values(MemberId id, const WCharSeq& value);
+
+  typedef Sequence<ACE_CDR::Octet> ByteSeq;
+  DDS::ReturnCode_t get_byte_values(ByteSeq& value, MemberId id) const;
+  DDS::ReturnCode_t set_byte_values(MemberId id, const ByteSeq& value);
+
+  typedef Sequence<ACE_CDR::Boolean> BooleanSeq;
+  DDS::ReturnCode_t get_boolean_values(BooleanSeq& value, MemberId id) const;
+  DDS::ReturnCode_t set_boolean_values(MemberId id, const BooleanSeq& value);
+
+  typedef Sequence<DCPS::String> StringSeq;
+  DDS::ReturnCode_t get_string_values(StringSeq& value, MemberId id) const;
+  DDS::ReturnCode_t set_string_values(MemberId id, const StringSeq& value);
+
+  typedef Sequence<DCPS::WString> WStringSeq;
+  DDS::ReturnCode_t get_wstring_values(WStringSeq& value, MemberId id) const;
+  DDS::ReturnCode_t set_wstring_values(MemberId id, const WStringSeq& value);
+
+  // Skip the whole part of the data stream that corresponds to this type.
+  // This is called by a containing type when it wants to skip a member which
+  // is an object of this type.
```

Comment:
```suggestion
  /// Skip the whole part of the data stream that corresponds to this type.
  /// This is called by a containing type when it wants to skip a member which
  /// is an object of this type.
```

Too make this Doxygen comments. Please do the rest with other comments like this.